libipl/p10/ipl0.C

```diff
@@ -439,6 +439,8 @@ static int ipl_sbe_config_update(void)
 	struct pdbg_target *root, *proc;
 	int rc = 1;
 	uint8_t istep_mode, core_mode, disable_security, attr_override;
+	uint8_t no_xscom_enforce;
+
```

Comment:
Can we avoid negative names? They are confusing to use and end up creating logical problems.

Reply:
Ack

Reply:
I also wouldn't call it "xscom" since it has nothing to do with xscoms.  The enforcement is for scoms issues by the SBE itself, which is via a different (not xscom) internal method.  The description of the ATTR_BOOT_FLAGS that this comes from is this:
  Bit 11    : Disable Scom white/blacklist in secure systems

Reply:
Thanks , updated. variable name.